client/my-sites/marketplace/index.js

```diff
@@ -34,9 +34,16 @@ export default function () {
 			clientRender
 		);
 		page(
-			'/marketplace/product/install/:site',
+			'/marketplace/product/install/:site?',
```

Comment:
Do we need this route? It should be covered by `'/marketplace/:productSlug?/install/:site?'` which also accepts the `:productSlug` parameter.

Reply:
Changing `page( `/marketplace/product/install/${ nextProps.siteSlug }` );` to `page( `/marketplace/install/${ nextProps.siteSlug }` );` should be enough to be picked up by `'/marketplace/:productSlug?/install/:site?'`

Reply:
Applied!